Backends/CLX/fonts.lisp

```diff
@@ -53,13 +53,21 @@
     ((:sans-serif (:bold :italic)) . "bold-o")
     ((:sans-serif (:italic :bold)) . "bold-o")))
 
+(clim-internals::define-protocol-class font-renderer ())
```

Comment:
do you plan to define font-renderer protocol? if yes, at least a reference manual for functions defining the protocol should be added.

Reply:
Sure, I can do that. Where do I put such documentation?